stats/view/aggregation_data.go

```diff
@@ -208,6 +229,26 @@ func (a *DistributionData) equal(other AggregationData) bool {
 	return a.Count == a2.Count && a.Min == a2.Min && a.Max == a2.Max && math.Pow(a.Mean-a2.Mean, 2) < epsilon && math.Pow(a.variance()-a2.variance(), 2) < epsilon
 }
 
+func (a *DistributionData) toPoint(metricType metricdata.Type, t time.Time) metricdata.Point {
+	buckets := []metricdata.Bucket{}
```

Comment:
nit: you can also check `metricType` here (expect to be cumulative distribution).